jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceListViewImpl.java

```diff
@@ -159,21 +161,25 @@ public void initColumns(final ListTable<ProcessInstanceSummary> extendedPagedTab
                                          constants.Correlation_Key()));
         columnMetas.add(new ColumnMeta<>(errorCountColumn,
                                          constants.Errors()));
-        columnMetas.add(actionsColumnMeta);
 
         List<GridColumnPreference> columnPreferenceList = extendedPagedTable.getGridPreferencesStore().getColumnPreferences();
 
-        for (GridColumnPreference colPref : columnPreferenceList) {
-            if (!isColumnAdded(columnMetas,
-                               colPref.getName())) {
-                Column genericColumn = initGenericColumn(colPref.getName());
-                genericColumn.setSortable(false);
-                columnMetas.add(new ColumnMeta<>(genericColumn,
-                                                 colPref.getName(),
-                                                 true,
-                                                 true));
+        columnPreferenceList.stream().filter(colPref -> !isColumnAdded(columnMetas, colPref.getName())).map(colPref -> {
+            String columnName = colPref.getName();
+            if (isSameColumn(colPref, columnMetas)) {
+                columnName = "Var_" + columnName;
             }
-        }
+            Column genericColumn = initGenericColumn(columnName);
+            genericColumn.setSortable(false);
+            columnMetas.add(new ColumnMeta<>(genericColumn,
```

Comment:
should you take advantage of the stream that you're using here and return the new ColumnMeta, the result is a collection of new columns that then could be added to columnMetas?